src/components/Images.vue

```diff
@@ -333,6 +336,9 @@ export default {
     imageTag(tag) {
       return tag === '<none>' ? 'latest' : `${ tag.trim() }`;
     },
+    isNotNoneImage(row) {
+      return row.imageName && row.imageName !== '' && row.imageName !== '<none>';
```

Comment:
We should be able to safely make use of optional chaining and simplify the expression a little bit 🙂

```suggestion
      return row?.imageName !== '' && row?.imageName !== '<none>';
```

Reply:
Thanks Phil! Just pushed.